v-next/example-project/contracts/Counter.t.sol

```diff
@@ -8,25 +8,31 @@ contract CounterTest {
   Counter counter;
 
   function setUp() public {
+    console.log("Setting up");
     counter = new Counter();
+    console.log("Counter set up");
```

Comment:
Are these changes being used somewhere? The only place where `Counter set up` shows up is in the formatters test, where the traces are hardcoded.

Reply:
I only used this for manual testing - to see how these logs would get printed out.